### DISCUSSION TITLE
封面信息隐去时出现问题

### DISCUSSION THREAD

**Discussion by shawchoy** [note286/xduts#182] [Q&A] [answered]
`\xdusetup{
%是否隐去个人信息
style/anonymous = true,
style / cjk-font = fandol,
style / latin-font = gyre,
% style/math-font=xits,
info/graduate-type = 硕士,
info/degree-type = 专业,
info/domain ={\anon[]{新一代电子信息技术（含量子技术等}},
info/degree ={电子信息},
info/degree* = {Electronic Information},
info/supervisor = {\anon[]{老师}},
info/supervisor* = {\anon[]{}},
info/supv-ent ={\anon[]{}},
info/supv-ent* ={\anon[]{}},
info/supv-title = {教授},
}`
![5 BWAPQ8C(F `JUCVDVJ4](https://github.com/note286/xduts/assets/94467157/0f8fccdb-3d00-426c-af8a-61955d0f805f)

**Comment by shawchoy** [comment]
导师的中英文姓名、职称隐去时都会出现上述问题

**Comment by note286** [comment, marked as answer]
https://github.com/note286/xduts/issues/61

**Reply by shawchoy** [reply]
了解了
